pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1447,17 +1447,6 @@ func getCPUTopology(vmi *v1.VirtualMachineInstance) *CPUTopology {
 		}
 	}
 
-	if vmiCPU == nil || (vmiCPU.Cores == 0 && vmiCPU.Sockets == 0 && vmiCPU.Threads == 0) {
```

Comment:
Update order right now is like this:

 1) virt-handlers
 2) virt-controllers (from that point in time on new virt-launchers are used)
 3) virt-api (which brings the new mutator)

Reply:
Therefore I think we have to keep this in place.

Reply:
How is the memory overcommit is being set in the mutator? 
https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator.go#L246

If we will keep this, we will have to calculate this again in the memory overhead function:
 https://github.com/kubevirt/kubevirt/blob/68f85fd17e68618542281b8e93050ef806d571c7/pkg/virt-controller/services/template.go#L1211

Reply:
Right now it is obviously wrong. What I mean is that, during an update we could end up with VMI definitions where the vmiCPU is not set, leading to a scenario where the converter would fail.

Reply:
@rmohr done :)